source/stylesheets/screen.css.scss

```diff
@@ -19,3 +19,27 @@
   border: 1px solid $error-colour;
   color: $error-colour;
 }
+
+.icinga-alert-backlink {
```

Comment:
Rather than giving this a name that is specific to it's use you could consider it's purpose, that way it can be reused across the manual if needed.

For example 'information-callout'